kuma/payments/jinja2/payments/payments.html

```diff
@@ -42,168 +42,212 @@ <h3>{{ title }}</h3>
+                        <li>{{_('Improving performance of the site')}}</li>
+                        <li>{{_('Modernizing the MDN platform')}}</li>
+                        <li>{{_('Adding offline access to MDN')}}</li>
+                        <li>{{_('Supporting integrations with popular developer tools')}}</li>
+                        <li>{{_('More tutorials and guides')}}</li>
+                        <li>{{_('Training and webinars')}}</li>
+                    </ul>
+                {% endcall %}
+                {%- call faq_entry(8, _("Why can’t you just open a Crowdsourcing campaign?")) %}
+                    <p>
+                        {% trans %}
+                            Because we aren’t looking for a lump sum. Our goal is
+                            to create a broad base of financial support from the
+                            people who benefit from the work of MDN.
+                        {% endtrans %}
+                    </p>
+                {% endcall %}
+                {%- call faq_entry(9, _("How has MDN been funded to date?")) %}
+                    <p>
+                        {% trans %}
+                            MDN is funded out of the Mozilla Corporation general
+                            budget (and has been since it was founded in 2005).
+                            Mozilla Corporation intends to continue to financially
+                            support MDN into the future, even as we broaden and
+                            diversify the sources of MDN funding. We just want to
+                            do more things with you and for you!
+                        {% endtrans %}
+                    </p>
+                {% endcall %}
+                {%- call faq_entry(10, _("How does Mozilla make money?")) %}
+                    <p>
+                        {% trans %}
+                            The Mozilla Corporation, which funds MDN, makes money
+                            primarily from royalties from search providers on
+                            Firefox (such as Google, Amazon, Duck Duck Go, and
+                            others).
+                        {% endtrans %}
+                    </p>
+                    <p>
+                        {% trans url="https://www.mozilla.org/en-US/foundation/annualreport/2016/" %}
+                            Separately, the Mozilla Foundation is a not-for-profit,
+                            making its money primarily from donations and royalties
+                            from Mozilla Corporation. As a not-for-profit, the
+                            Mozilla Foundation reports these revenues publicly
+                            every trailing year, as in our most recent
+                            <a href="{{ url }}">2016 Mozilla Annual Report</a>.
+                        {% endtrans %}
+                    </p>
+                {% endcall %}
+
+
```

Comment:
Nit: two lines of un-needed


 whitespace

Reply:
done